### PR TITLE
feature-stop-listing-expired-product-8

### PR DESCRIPTION
**Pull request by vanessa-abewe**
**What does this PR do?**

Expired Products should never appear on the list of the available products

 **Description of Task to be completed?**

Given the expiration date has passed, the product should be removed from the list of available products

What are the relevant pivotal tracker/Trello stories? 
Trello Card : https://trello.com/c/wdswB4hf

### CODE REVIEW

**Review by ALVINdimpos — APPROVED** [review]

**Review by yobuir — APPROVED** [review]

File: src/app.js [hunk 1]
```diff
@@ -79,5 +79,11 @@ app.use('/api/category', category);
 app.use('/api', wishlistRoute);
 app.use("/", welcomeRoute); 
 app.use("/api/cart", cartRoute);
+app.use('/api/category', category);
+app.use('/api', wishlistRoute);
```
Comment (chanceDira) [inline src/app.js]: This line is duplicated

File: src/controller/expiredProductController.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    }
+    
+    for (const product of products) {
+      const productExpiryDate = product.expiryDate;
+      
+      if (productExpiryDate < currentDate) {
+        const seller = await db.User.findOne({ where: { id: product.vendor_id } });
+        
+        if (!seller) {
+          continue;
```
Comment (chanceDira) [inline src/controller/expiredProductController.js:26]: Here let's have seller not found

**Review by chanceDira — APPROVED** [review]
Nice work @Vanessa-bitega 

I just left some feedback, please also add commands to test your work

**Review by chanceDira — APPROVED** [review]
Good job @Vanessa-bitega

**Pull request merged**